airbyte-integrations/bases/base-java-s3/src/main/java/io/airbyte/integrations/destination/s3/GlueOperations.java

```diff
@@ -0,0 +1,156 @@
+package io.airbyte.integrations.destination.s3;
+
+import com.amazonaws.services.glue.AWSGlue;
+import com.amazonaws.services.glue.model.Column;
+import com.amazonaws.services.glue.model.CreateTableRequest;
+import com.amazonaws.services.glue.model.DeleteTableRequest;
+import com.amazonaws.services.glue.model.EntityNotFoundException;
+import com.amazonaws.services.glue.model.GetTableRequest;
+import com.amazonaws.services.glue.model.SerDeInfo;
+import com.amazonaws.services.glue.model.StorageDescriptor;
+import com.amazonaws.services.glue.model.TableInput;
+import com.amazonaws.services.glue.model.UpdateTableRequest;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.base.Preconditions;
+import java.io.IOException;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.function.UnaryOperator;
+import java.util.stream.Collectors;
+
+//TODO (itaseski) implement wrapper for retry logic on transient errors
+public class GlueOperations implements MetastoreOperations {
+
+    private final ObjectMapper objectMapper;
+
+    private final AWSGlue awsGlueClient;
+
+    public GlueOperations(AWSGlue awsGlueClient) {
+        Preconditions.checkArgument(awsGlueClient != null);
+        this.awsGlueClient = awsGlueClient;
+        this.objectMapper = new ObjectMapper();
+    }
+
+    //TODO (itaseski) can location change after table is created?
```

Comment:
That's a good question. I would imagine the answer is yes? In practice I think it's a rare occurrence.

Reply:
I have provided the ability to update the location through the input parameter so we are good to go.